modules/dr-restoring-cluster-state.adoc

```diff
@@ -51,641 +55,53 @@ For non-recovery control plane nodes, it is not required to establish SSH connec
-+
-[source,terminal]
-----
-NAME                                        PHASE          TYPE        REGION      ZONE         AGE     NODE                           PROVIDERID                              STATE
-clustername-8qw5l-master-1                  Running        m4.xlarge   us-east-1   us-east-1b   3h37m   ip-10-0-143-125.ec2.internal   aws:///us-east-1b/i-096c349b700a19631   running
-clustername-8qw5l-master-2                  Running        m4.xlarge   us-east-1   us-east-1c   3h37m   ip-10-0-154-194.ec2.internal    aws:///us-east-1c/i-02626f1dba9ed5bba  running
-clustername-8qw5l-master-3                  Provisioning   m4.xlarge   us-east-1   us-east-1a   85s     ip-10-0-173-171.ec2.internal    aws:///us-east-1a/i-015b0888fe17bc2c8  running <1>
-clustername-8qw5l-worker-us-east-1a-wbtgd   Running        m4.large    us-east-1   us-east-1a   3h28m   ip-10-0-129-226.ec2.internal   aws:///us-east-1a/i-010ef6279b4662ced   running
-clustername-8qw5l-worker-us-east-1b-lrdxb   Running        m4.large    us-east-1   us-east-1b   3h28m   ip-10-0-144-248.ec2.internal   aws:///us-east-1b/i-0cb45ac45a166173b   running
-clustername-8qw5l-worker-us-east-1c-pkg26   Running        m4.large    us-east-1   us-east-1c   3h28m   ip-10-0-170-181.ec2.internal   aws:///us-east-1c/i-06861c00007751b0a   running
-----
-<1> The new machine, `clustername-8qw5l-master-3` is being created and is ready after the phase changes from `Provisioning` to `Running`.
-+
-It might take a few minutes for the new machine to be created. The `etcd` cluster Operator will automatically sync when the machine or node returns to a healthy state.
-
-.. Repeat these steps for each lost control plane host that is not the recovery host.
-
-. Turn off the quorum guard by entering:
-+
-[source,terminal]
-----
-$ oc patch etcd/cluster --type=merge -p '{"spec": {"unsupportedConfigOverrides": {"useUnsupportedUnsafeNonHANonProductionUnstableEtcd": true}}}'
-----
-+
-This command ensures that you can successfully re-create secrets and roll out the static pods.
-
-. In a separate terminal window within the recovery host, export the recovery `kubeconfig` file by running:
-+
-[source,terminal]
-----
-$ export KUBECONFIG=/etc/kubernetes/static-pod-resources/kube-apiserver-certs/secrets/node-kubeconfigs/localhost-recovery.kubeconfig
-----
-
-. Force `etcd` redeployment.
-+
-In the same terminal window where you exported the recovery `kubeconfig` file, run:
-+
-[source,terminal]
-----
-$ oc patch etcd cluster -p='{"spec": {"forceRedeploymentReason": "recovery-'"$( date --rfc-3339=ns )"'"}}' --type=merge <1>
-----
-<1> The `forceRedeploymentReason` value must be unique, which is why a timestamp is appended.
-+
-The `etcd` redeployment starts.
-+
-When the `etcd` cluster Operator performs a redeployment, the existing nodes are started with new pods similar to the initial bootstrap scale up.
-
-. Turn the quorum guard back on by entering:
++ Once recovered, enable the quorum guard by running:
```

Comment:
Extra +

Reply:
Oops, that was a typo. Glad you noticed it!